go.mod

```diff
@@ -67,14 +67,13 @@ require (
 	go.uber.org/atomic v1.9.0 // indirect
 	go.uber.org/multierr v1.7.0 // indirect
 	go.uber.org/zap v1.21.0 // indirect
-	golang.org/x/crypto v0.14.0 // indirect
-	golang.org/x/mod v0.9.0 // indirect
-	golang.org/x/net v0.17.0 // indirect
+	golang.org/x/crypto v0.24.0 // indirect
+	golang.org/x/net v0.26.0 // indirect
```

Comment:
This is actually higher than what we have in `main` and `release-v0.13`: https://github.com/shipwright-io/operator/blob/release-v0.13/go.mod#L74. This could lead to a regression during upgrade.

Since v0.25.0 also mitigates the reported vulnerability, please change the bump to reference v0.25.0 instead.